Makefile

```diff
@@ -2,22 +2,42 @@ UNAME:=$(shell uname)
 
 # Install settings
 PREFIX= /opt/cartesi
-BIN_INSTALL_PATH= $(PREFIX)/bin
-LIB_INSTALL_PATH= $(PREFIX)/lib
-SHARE_INSTALL_PATH= $(PREFIX)/share
-IMAGES_INSTALL_PATH= $(SHARE_INSTALL_PATH)/images
-UARCH_INSTALL_PATH= $(SHARE_INSTALL_PATH)/uarch
-LUA_INSTALL_CPATH= $(PREFIX)/lib/lua/5.4
-LUA_INSTALL_PATH= $(PREFIX)/share/lua/5.4
-INC_INSTALL_PATH= $(PREFIX)/include/machine-emulator
+MACHINE_EMULATOR_VERSION:= $(shell make -sC src get-version)
+MACHINE_EMULATOR_SO_VERSION:= $(shell make -sC src get-so-version)
+DEB_FILENAME= cartesi-machine-v$(MACHINE_EMULATOR_VERSION).deb
+BIN_RUNTIME_PATH= $(PREFIX)/bin
+LIB_RUNTIME_PATH= $(PREFIX)/lib/cartesi-machine
```

Comment:
I noticed in recent changes that we don't need to provide grpc libraries anymore. With the change the `cartesi-machine` folder in `lib` becomes less necessary. What do you think about removing it? So we install `$(PREFIX)/lib/libcartesi.so` instead of `$(PREFIX)/lib/cartesi-machine/libcartesi.so`